app/cdap/components/Connections/AddConnectionBtnModal/index.tsx

```diff
@@ -37,7 +37,7 @@ export default function AddConnectionBtnModal({ onCreate, className = null }) {
   }
 
   function onCreateHandler() {
-    setIsOpen(false);
+    // setIsOpen(false);
```

Comment:
do we want to comment this out?

Reply:
They should be useless so I removed them. Both buttons in namespace and wrangler work properly.